alicloud/resource_alicloud_cen_bandwidth_package_test.go

```diff
@@ -0,0 +1,224 @@
+package alicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/cbn"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
```

Comment:
This resource test should supports sweeper. Please refer to https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/resource_alicloud_cen_instance_test.go#L16